### ISSUE TITLE
md-scroll-shrink only checks for attribute not value 

### ISSUE THREAD

**Issue by langan** [angular/material#4363]
Trying to turn on and off scroll shrink using a variable.

Like this

```HTML
<md-toolbar md-scroll-shrink="scrollShrinkOn">
```

But the attribute doesn't look like it's being watched.

Even

```HTML
<md-toolbar md-scroll-shrink="false">
```

Doesn't work [the docs](https://material.angularjs.org/latest/#/api/material.components.toolbar/directive/mdToolbar) say it should be boolean.

But looking at the code it just checks if the attribute is defined?

```JS
      if (angular.isDefined(attr.mdScrollShrink)) {
        setupScrollShrink();
      }
```

Would be happy to look into a fix if needed

**Comment by topherfangio** [comment]
This sounds like a fantastic community contribution if you're willing to put together a PR!

General principle is:

1. Create a test that shows the feature you are implementing (it should look something like [this](https://github.com/angular/material/blob/master/src/components/fabSpeedDial/fabSpeedDial.spec.js#L31]))
1. Change the code to make the test pass
1. Add the proper docs to `toolbar.js`
1. Run all tests and make sure they pass
1. Submit the PR :-)

Please let us know if you have any questions along the way!

**Comment by ThomasBurleson** [comment]
Fixed with SHA 9b861fdb7d9de924e6549f4e2b2bf07004462295
